docs/relnotes.md

```diff
@@ -30,6 +30,9 @@ Not published yet.
 3.0.0
 
 - `Scope.PIPELINE` has been removed from `Scope^` values.
+- The `root_folder`, `storage_folder`, `read_entity_retry`, `repository_type`, and `repository_properties`
```

Comment:
```suggestion
- The `root_folder`, `storage_folder`, `repository_type`, and `repository_properties`
```

Reply:
The `read_entity_retry` was on `GlobalAppConfig` before. It was also documented, so I think we should specify it here.

Reply:
Absolutely.